pkg/horizon/deployment.go

```diff
@@ -109,6 +109,22 @@ func Deployment(
 				Spec: corev1.PodSpec{
 					ServiceAccountName: instance.RbacResourceName(),
 					Containers: []corev1.Container{
+						// the first container in a pod is the default selected
+						// by oc log so define the log stream container first.
+						{
+							Name: instance.Name + "-log",
```

Comment:
you need to update envTests to point to container[1] when checking horizon, and not the sidecar w/ logs.

```
--- a/tests/functional/horizon_controller_test.go
+++ b/tests/functional/horizon_controller_test.go
@@ -580,7 +580,7 @@ var _ = Describe("Horizon controller", func() {
                        th.AssertVolumeExists(CABundleSecretName, d.Spec.Template.Spec.Volumes)
                        th.AssertVolumeExists(InternalCertSecretName, d.Spec.Template.Spec.Volumes)

-                       svcC := d.Spec.Template.Spec.Containers[0]
+                       svcC := d.Spec.Template.Spec.Containers[1]

                        // check TLS volume mounts
                        th.AssertVolumeMountExists(CABundleSecretName, "tls-ca-bundle.pem", svcC.VolumeMounts)
```